packages/nextjs/test/sample.test.ts

```diff
@@ -0,0 +1,5 @@
+describe('sample test', () => {
+  test('true is truthy', () => {
+    expect(true).toBeTruthy();
+  });
+});
```

Comment:
This file seems unrelated to the contents of this PR

Reply:
Deleting `initDecider` also means deleting its test, which was the only test. CI fails when there are no tests ([see failed build](https://github.com/getsentry/sentry-javascript/actions/runs/708130998), `@sentry/nextjs: No tests found, exiting with code 1`); that's why I added this sample test. Maybe we should add `--passWithNoTests`? Wdyt?

Reply:
Aha, I get it... no tests.

Reply:
> Maybe we should add --passWithNoTests? Wdyt?

Normally I'd say yes, but we should def add tests before we're through with this whole thing, so I think it's fine to leave the placeholder.